src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -405,7 +405,7 @@ const UsersFragment = ({
         data={peopleYouMightKnowData || []}
         renderItem={(props) => renderItem({...props, section: 'suggested-user'})}
         keyExtractor={(item, index) => index.toString()}
-        onEndReached={() => fetchData()}
+        onEndReached={() => (initialAccordionData.length > 0 ? fetchData() : null)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good one as it adds a condition to check if `initialAccordionData` has any data before calling `fetchData()`. This can prevent unnecessary network requests when there's no more data to fetch. However, it would be better to also check if `fetchData` is not null or undefined before calling it to avoid potential errors.
```diff
- onEndReached={() => (initialAccordionData.length > 0 ? fetchData() : null)}
+ onEndReached={() => (initialAccordionData.length > 0 && fetchData ? fetchData() : null)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->